Add integration test of SyntaxAnalyzeExt ET

hellozepp opened the pull request:
1. Fix default value of SyntaxAnalyzeExt parameter
2. Add integration test of SyntaxAnalyzeExt ET

allwefantasy commented on streamingpro-it/src/test/resources/sql/simple/09_syntax_analysis_ext_test.mlsql:
```diff
@@ -0,0 +1,8 @@
+select 1 as a as b;
```
Please use !assert instead of `expected`

allwefantasy commented on streamingpro-it/src/test/scala/tech/mlsql/it/LocalBaseTestSuite.scala:
```diff
@@ -54,7 +54,8 @@ trait LocalBaseTestSuite extends FunSuite with SparkOperationUtil with BeforeAnd
     copyDataToUserHome(user)
     TestManager.loadTestCase(new File(testCaseDirPath))
-    PlatformManager.getOrCreate.run(new ParamsUtil(runParams))
+    // Load built-in and external plug-ins and start PlatformManager
```
Remove this since PR https://github.com/allwefantasy/mlsql/pull/1511  have resolved this issue already.

allwefantasy commented on external/mlsql-ets/src/main/java/tech/mlsql/plugins/ets/SyntaxAnalyzeExt.scala:
````diff
@@ -96,55 +91,68 @@ class SyntaxAnalyzeExt(override val uid: String) extends SQLAlg with WowParams w
+      | First generate two tables, table1 and table2. Then execute SyntaxAnalyzeExt to
+      | parse a nested SQL. As follows:
+      | ```sql
+      | select "stub" as table1;
````
This two lines :

1. Have wrong syntax.
2. No need stub table to make your ET works.

allwefantasy commented on external/mlsql-ets/src/main/java/tech/mlsql/plugins/ets/SyntaxAnalyzeExt.scala:
```diff
@@ -96,55 +91,68 @@ class SyntaxAnalyzeExt(override val uid: String) extends SQLAlg with WowParams w
     """
-      |Required. SQL to be analyzed
-      |e.g. sql = "select * from table"
+      | Required. SQL to be analyzed
```
Also you should issue a new PR to fix the doc in https://github.com/allwefantasy/mlsql-docs/commit/204a8ad1daf730991ccf99169831f52a34379547

allwefantasy committed:
run-test.sh support 3.1.1

update

update

update

update

update

update

update

update

update

update

update

update

update

hellozepp committed:
1. Fix default value of SyntaxAnalyzeExt parameter
2. Add integration test of SyntaxAnalyzeExt ET
3. Use !assert instead of expected
4. remove useless stub table

allwefantasy committed:
run-script.sh support skipInstall
